ui/component/or-mwc-components/src/or-mwc-input.ts

```diff
@@ -665,6 +667,9 @@ export class OrMwcInput extends LitElement {
     @property({type: Boolean})
     public autoSelect?: boolean;
 
+    @property({type: Boolean})
+    public eventOnInput?: boolean;
+
```

Comment:
The properties of this component generally replicate the standard HTML input attributes; for a particular instance of `or-mwc-input` you can just attach event handlers that you want:

```
<or-mwc-input @input=.....
```

Reply:
Hi @richturner , for this, using @input force me to leave to focus on the input field before any change has been "returned" to the asset-tree listening for input.
But if in or-mwc-input, I also add @input on inputElement (in addition to @change and with the same event handling), then it's working.